crates/editor/src/editor.rs

```diff
@@ -3907,6 +3920,61 @@ impl Editor {
         }))
     }
 
+    pub fn show_signature_help(&mut self, _: &ShowSignatureHelp, cx: &mut ViewContext<Self>) {
+        if self.pending_rename.is_some() {
+            return;
+        }
+
+        let position = self.selections.newest_anchor().head();
+        let Some((buffer, buffer_position)) =
+            self.buffer.read(cx).text_anchor_for_position(position, cx)
+        else {
+            return;
+        };
+
+        let task = cx.spawn(move |this, mut cx| async move {
```

Comment:
```suggestion
        self.signature_help_task = Some(cx.spawn(move |this, mut cx| async move {
```

NIT style remark, and feel free to just resolve the issue when it's fixed, with no replies.

Reply:
Thanks!
I fixed it on 4b5dbc4d0